hazelcast/src/main/java/com/hazelcast/sql/impl/schema/map/MapTableUtils.java

```diff
@@ -75,6 +75,10 @@ public static long estimatePartitionedMapRowCount(NodeEngine nodeEngine, MapServ
     public static List<MapTableIndex> getPartitionedMapIndexes(MapContainer mapContainer, List<TableField> fields) {
         Map<QueryPath, Integer> pathToOrdinalMap = mapPathsToOrdinals(fields);
 
+        if (mapContainer.getIndexes() == null) {
```

Comment:
shouldn't we try to get indexes from partitions here?
like in https://github.com/hazelcast/hazelcast/blob/master/hazelcast/src/main/java/com/hazelcast/map/impl/MapContainer.java#L338

Reply:
@promanenko This method is supposed to find local indices. SQL reads only global indices.